src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -40,16 +40,17 @@ internal class InternalXrefMapBuilder
             _jsonSchemaTransformer = jsonSchemaTransformer;
         }
 
-        public IReadOnlyDictionary<string, InternalXrefSpec[]> Build()
+        public (IReadOnlyDictionary<string, InternalXrefSpec[]>, IReadOnlyList<(string, SourceInfo?)>) Build()
         {
             var builder = new ListBuilder<InternalXrefSpec>();
+            var globalUIDs = new ListBuilder<(string, SourceInfo?)>();
```

Comment:
>globalUIDs [](start = 16, length = 10)

internalXrefMap and externalXrefMap should be enough for this feature

Reply:
OK